portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,69 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Sana Jain</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+<body>
+
+<div class = "header">
+    <h1> SANA JAIN </h1>
+</div>
+
+<div class="topnav">
+  <a href="https://www.linkedin.com/in/sanajain/">LinkedIn</a>
+  <a href="mailto:jainsana99@gmail.com">Email</a>
+  <a href="https://www.instagram.com/sana_jain21">Instagram</a>
+</div>
+
+<div class = "row">
+    <div class = "leftcolumn"> 
+        <div class = "card">
+            <h2> ABOUT ME </h2>
+            <p> Welcome! I'm Sana and I am currently studying Nanotechnology Engineering at the University of Waterloo. I am most interested in the applications of nanotechnology in the fields of electrical and material engineering. My past work opportunities have been quite diverse and unique. My experiences range from working in a research lab in Germany to interning at an engineering consulting firm in Toronto and that's just the start. Keep scrolling to learn even more about my experiences and interests! </p>
+        </div>
+        <div class = "card"> 
+            <h2> KARLSRUHE INSTITUTE OF TECHNOLOGY </h2>
+            <p> Working abroad in Germany for four months was a fantastic experience. Not only did I gain technical skills, as I researched methods of printing Field Effect Transistors (FETs), but I had the opportunity to travel across Europe. I saw over 12 countries, gained hands on experience, and found my interest for electrical engineering. This internship was a dream!</p>
+        </div> 
+        <div class = "card">
+            <h2> ARUP ENGINEERING CONSULTING FIRM </h2>
+            <p> At Arup I had the opportunity to learn so much about designing the electrical systems of a building or infastructure project! I gained experience in designing power distribution systems, lighting, and fire alarms for large scale projects. I utilized softwares such as Revit and Microstation in order to create these designs. Some of the projects I worked on included: 
+                <ul class="feature-icons">
+					<li class="icon fa-plane">Toronto Pearson International Airport</li>
+					<li class="icon fa-train">Finch West and Edmonton LRT</li>
+					<li class="icon fa-gear">Canadian Canoe Museum</li>
```

Comment:
What do you think about making these < li > items a single indent more than the < ul >?

Reply:
Good catch, thank you for pointing that out! I have updated the html file to address these changes.